app/components/MealsList .jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { useEffect } from "react";
+
+export default function MealsList() {
+  const [meals, setMeals] = useState([]);
+
+  useEffect(() => {
+    const handleData = async () => {
+      const response = await fetch("http://localhost:3000/all-meals");
+
+      const res = await response.json();
+      setMeals(res);
+    };
+
+    handleData();
+  }, []);
+
+  {
+    meals.map((meal) => {
+      return (
+        <div>
+          <h3> title: {meal.title} </h3>
+          <p>description: {meal.description}</p>
+          <p>price: {meal.price}</p>
+        </div>
+      );
+    });
+  }
+}

```

Comment:
Is anything being returned from MealsList? You're rendering inside a .map() function, but not returning anything from the component itself.